### PR TITLE
Add prompt string interpolation

### PR DESCRIPTION
**Pull request by pederhan**
Adds string interpolation support for the CLI prompt. In order to test this, the `get_prompt_message()` function has been extracted from `main()`. 

Furthermore, the parsing of command-line config arguments is now moved into the config itself. CLI args are now accessed through `MregCliConfig.get()` instead of via `vars(args)`.

### CODE REVIEW

**Review by terjekv — APPROVED** [review]
LGTM!

File: mreg_cli/main.py [hunk 1]
```diff
@@ -241,5 +214,46 @@ def get_prompt_message():
...
+            host=host,
+        )
+
+    # Try to get prompt from args -> config -> default
+    if (args_prompt := args_map.get("prompt")) is not None:
+        prompt = str(args_prompt)
+    elif (config_prompt := config.get_prompt()) is not None:
+        prompt = config_prompt
+    else:
+        prompt = DEFAULT_PROMPT
```
Comment (pederhan) [inline mreg_cli/main.py:238]: This doesn't look _great_, but we have to guard against both `args.prompt` and `config["prompt"]` being None.

File: README.md [hunk 2]
```diff
@@ -38,8 +38,49 @@ Options can be set in ~/.config/mreg-cli.conf. A typical config file looks like
...
+
+The prompt can be specified as a command line argument as well:
+
+```bash
+mreg-cli --prompt "<string>"
+```
+
+#### Disabling the prompt text
+
+The prompt text can be disabled by setting it to an empty string in either the config or as a command line argument:
```
[the remark this answers is not in the document]
Comment (pederhan) [reply README.md:75]: It means `>`, which is why I specified "prompt _text_". If you have any ideas how to make it clearer, then I'll change it.

File: mreg_cli/main.py (hunk 1 above)
Comment (pederhan) [reply mreg_cli/main.py:238]: Could call it "prefix" or something.

**Comment by pederhan** [comment]
I _think_ this is ready now? There are some basic unit test for IPv4 and IPv6, but they are not comprehensive at all. Should be fine though.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pederhan) [reply README.md]: This was changed to "prompt prefix text", which isn't great, but it should remove any ambiguity when combined with the example showing the result.

**Comment by pederhan** [comment]
One last thing we should consider taking a look at is moving the prompt configuration section in the README somewhere else, because it takes up a lot of space very early on for such a trivial feature.

File: README.md [hunk 3]
```diff
@@ -38,10 +38,65 @@ Options can be set in ~/.config/mreg-cli.conf. A typical config file looks like
...
+
+### Prompt
+
+The prompt text can be configured with a custom format string. The available variables are:
+
+- `{user}`: Username of active user
+- `{proto}`: Protocol part of the server URL
+- `{host}`: Name part of the server URL
+- `{port}`: Port part of the server URL (if any)
+- `{domain}`: Domain name. Defaults to `uio.no` if not specified.
```
Comment (terjekv) [inline README.md:56]: This was dropped?

Comment (terjekv) [reply README.md:56]: Ah, right, fun! Thanks!

**Review by terjekv — APPROVED** [review]
LGTM!

**Pull request merged**